src/app/components/search/search.html

```diff
@@ -9,6 +9,21 @@
     .sub-results {
         margin-left: 36px;
     }
+    .line-clamp-2 {
+	   overflow: hidden;
+	   text-overflow: ellipsis;
+	   display: -webkit-box;
+	   -webkit-line-clamp: 2; 
+	   -webkit-box-orient: vertical;
+	}
+    .line-clamp-1 {
+	   overflow: hidden;
+	   text-overflow: ellipsis;
+	   display: -webkit-box;
+	   -webkit-line-clamp: 1; 
```

Comment:
It feels like there should be some way to extend a base line-clamp, but my CSS isn't good enough so just copy-pasting for now.